Nodejs/Product/Nodejs/Guids.cs

```diff
@@ -33,8 +33,9 @@ static class Guids
         public const string NodejsBaseProjectFactoryString = "9092AA53-FB77-4645-B42D-1CCCA6BD08BD";
         public const string TypeScriptLanguageInfoString = "87bdf188-e6e8-4fcf-a82a-9b8506e01847";
         public const string JadeEditorFactoryString = "6CB69EF8-1329-4DC0-84B4-FA134EA59BE3";
+        public const string DefaultLanguageServiceString = "{8239BEC4-EE87-11D0-8C98-00C04FC2AB22}";
 
-        internal static readonly Guid DefaultLangaugeService = new Guid("{8239BEC4-EE87-11D0-8C98-00C04FC2AB22}");
+        internal static readonly Guid DefaultLanguageService = new Guid(DefaultLanguageServiceString);

```

Comment:
You refer to this guid in a bunch of places, but I'm not sure that's really the one you want. Essentially the default language service (JSLS) handles javascript completions within VS by default. In order to work around some of the performance issues caused by JSLS, we open javascript files in our own Node editor using our own language service (represented by the ```NodejsLanguageInfoString``` above. So if you want snippets to appear in the Node editor, you should be referencing the Nodejs string rather than the Default.